File: 비주얼 베이직/중간고사/1/1/Form1.vb
Public Class Form1
    Private Sub TextBox1_TextChanged(sender As Object, e As EventArgs) Handles TextBox1.TextChanged
        Dim i, N, oddsum, Evensum As Integer

        N = TextBox1.Text
        TextBox2.Text = ""
        TextBox3.Text = ""
        For i = 1 To N Step 1
            If (i Mod 2 = 1) Then
                oddsum = oddsum + i
                If (TextBox2.Text = "") Then
                    TextBox2.Text = TextBox2.Text & i
                Else
                    TextBox2.Text = TextBox2.Text & " + " & i
                End If

            Else
                Evensum = Evensum + i
                If (TextBox3.Text = "") Then
                    TextBox3.Text = TextBox3.Text & i
                Else
                    TextBox3.Text = TextBox3.Text & " + " & i
                End If
            End If
        Next
        TextBox2.Text = TextBox2.Text & " = " & oddsum
        TextBox3.Text = TextBox3.Text & " = " & Evensum
    End Sub
End Class

File: C#/2019-2-C샵-2주차-두숫자의연산-김유빈/2019-2-C샵-2주차-두숫자의연산-김유빈/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace _2019_2_C샵_2주차_두숫자의연산_김유빈
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        private void Form1_Load(object sender, EventArgs e)
        {
        }
        double Digit1, Digit2; //전역 변수
        private void button1_Click(object sender, EventArgs e)
        {           
            Digit1 = double.Parse(textBox1.Text);
            Digit2 = double.Parse(textBox2.Text);
            textBox3.Text = (Digit1 + Digit2).ToString();
            label4.Text = "+";
        }

        private void button2_Click(object sender, EventArgs e)
        {           
            Digit1 = double.Parse(textBox1.Text);
            Digit2 = double.Parse(textBox2.Text);
            textBox3.Text = (Digit1 - Digit2).ToString();
            label4.Text = "-";
        }

        private void button3_Click(object sender, EventArgs e)
        {
            Digit1 = double.Parse(textBox1.Text);
            Digit2 = double.Parse(textBox2.Text);
            textBox3.Text = (Digit1 * Digit2).ToString();
            label4.Text = "X";
        }

        private void button5_Click(object sender, EventArgs e)
        {
            Digit1 = double.Parse(textBox1.Text);
            Digit2 = double.Parse(textBox2.Text);
            textBox3.Text = (Digit1 % Digit2).ToString();
            label4.Text = "mod";
        }

        private void button6_Click(object sender, EventArgs e)
        {
            Digit1 = double.Parse(textBox1.Text);
            Digit2 = double.Parse(textBox2.Text);
            textBox3.Text = Math.Pow(Digit1 , Digit2).ToString();
            label4.Text = "^";
        }

        private void button4_Click(object sender, EventArgs e)
        {
            Digit1 = double.Parse(textBox1.Text);
            Digit2 = double.Parse(textBox2.Text);
            textBox3.Text = (Digit1 / Digit2).ToString();
            label4.Text = "÷";
        }
    }
}
